docs/source/notes/get_start_xpu.rst

```diff
@@ -8,64 +8,43 @@ Hardware Prerequisite
    :widths: 50 50
    :header-rows: 1
 
-   * - Validated Hardware
-     - Supported OS
-   * - Intel® Data Center GPU Max Series
-     - Linux
-   * - Intel Client GPU
-     - Windows/Linux
+   * - Supported OS
+     - Validated Hardware
+   * - Linux
+     - Intel® Data Center GPU Max Series/ Intel® Client GPUs (Intel® Arc™ B-Series graphics not supported)
+   * - Windows
+     - Intel® Client GPUs (Intel® Arc™ B-Series graphics with experimental support)
+   * - WSL2 (experimental feature)
+     - Intel® Client GPUs (Intel® Arc™ B-Series graphics/Intel® Core™ Ultra Series 2 with Intel® Arc™ Graphics not supported)
 
-Intel GPUs support (Prototype) is ready in PyTorch* 2.5 for Intel® Data Center GPU Max Series and Intel® Client GPUs on both Linux and Windows, which brings Intel GPUs and the SYCL* software stack into the official PyTorch stack with consistent user experience to embrace more AI application scenarios.
+Intel GPUs support (Prototype) is ready in PyTorch* 2.6 for Intel® Data Center GPU Max Series and Intel® Client GPUs on both Linux and Windows, which brings Intel GPUs and the SYCL* software stack into the official PyTorch stack with consistent user experience to embrace more AI application scenarios.
 
 Software Prerequisite
 ---------------------
 
-Visit `PyTorch Installation Prerequisites for Intel GPUs <https://www.intel.com/content/www/us/en/developer/articles/tool/pytorch-prerequisites-for-intel-gpus.html>`_ for more detailed information regarding:
-
-#. Intel GPU driver installation
-#. Intel support package installation
-#. Environment setup
+Visit `PyTorch Installation Prerequisites for Intel GPUs <https://www.intel.com/content/www/us/en/developer/articles/tool/pytorch-prerequisites-for-intel-gpus.html>`_ Driver Installation.
```

Comment:
It is better to provide the link to `Driver Installation` directly.

Reply:
yes, I will replace the link to the latest published one with driver installation session.